rfcs/0000-template.md

```diff
@@ -0,0 +1,67 @@
+---
+Start Date: DD-MM-YYYY
+RFC PR: (leave this empty if no PR yet)
+Blade Issue: (leave this empty if no issue yet)
+---
+
+# Title of the RFC <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Basic Example](#basic-example)
+- [Motivation](#motivation)
+- [Detailed Design](#detailed-design)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+- [Adoption strategy](#adoption-strategy)
+- [How do we educate people?](#how-do-we-educate-people)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Brief explanation about the proposal.
+
+# Basic Example
+Include a basic code example. Omit this section if it's not applicable.
+
+# Motivation
+- Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+Try to focus on explaining the motivation so that if this RFC is not accepted, the motivation could be used to develop alternative solutions. In other words, try to list down the constraints you are trying to solve without coupling them too closely to the solution you have in mind.
```

Comment:
Is this section about the solution or the motivation? Since it says:
> try to list down the constraints you are trying to solve without coupling them too closely to the solution you have in mind.

and then here it says
> What use cases does it support?

Seems confusing to me.

Reply:
This section is about motivation of why do we even need to do this? What is this solving? It's not about how to solution, it's about the end result and what it would solve.

Reply:
So the section says try to talk about motivation and philosophies so even if the RFC is not accepted we can still use some ideas from this section. Hence it says
>In other words, try to list down the constraints you are trying to solve without coupling them too closely to the solution you have in mind.